python/ray/autoscaler/_private/util.py

```diff
@@ -98,9 +98,28 @@ def fillout_defaults(config: Dict[str, Any]) -> Dict[str, Any]:
     defaults = _get_default_config(config["provider"])
     defaults.update(config)
     defaults["auth"] = defaults.get("auth", {})
+    try:
+        defaults = _fillout_available_node_types_resources(defaults)
+    except Exception:
+        # We don't want to introduce new errors with filling available node
+        # types resources feature.
```

Comment:
Let's print the error and a warning since this is an easy place to not have the right permissions.